src/gui/MainWindow.cpp

```diff
@@ -1176,7 +1176,14 @@ void MainWindow::ShowCursor(bool state)
 		} while (counter >= 0);
 	}
 	#else
-	cemuLog_log(LogType::Force, "MainWindow::ShowCursor - todo");
+	if (state)
```

Comment:
This actually looks like a portable solution.
Maybe we don't need this windows version that relies on the windows API.